tox.ini

```diff
@@ -0,0 +1,27 @@
+[tox]
+envlist = py27,py34,py35,flake8,cov
+
+[testenv]
+commands = py.test {posargs}
+basepython =
+    py27: python2.7
+    py34: python3.4
+    py35: python3.5
+    flake8: python2.7
+    cov: python2.7
```

Comment:
No need to run conditional coverage. Tests are so fast, that coverage should be run by default on all tests suite IMHO. No?